tracer/src/Datadog.Tracer.Native/calltarget_tokens.cpp

```diff
@@ -468,6 +471,16 @@ HRESULT CallTargetTokens::ModifyLocalSig(ILRewriter* reWriter, TypeSignature* me
     memcpy(&newSignatureBuffer[newSignatureOffset], &exTypeRefBuffer, exTypeRefSize);
     newSignatureOffset += exTypeRefSize;
 
+    // Exception value for calltarget exception filters
+    newSignatureBuffer[newSignatureOffset++] = ELEMENT_TYPE_CLASS;
+    memcpy(&newSignatureBuffer[newSignatureOffset], &exTypeRefBuffer, exTypeRefSize);
+    newSignatureOffset += exTypeRefSize;
```

Comment:
Apparently we can't, also this index is for a complete different exception, the exception that would have happened in the original code and that we should rethrow and can examine in the integration.
But I've still tried it, also tried to use one extra index only and not two for the 2 filter parts (begin / end method), everytime we end up with a broken c# code. The IL is correct though but the decompiled code is wrong, and again, the catch handler never gets called where the part is wrong and no exception is logged. 
![image](https://user-images.githubusercontent.com/8353486/211776058-728a0ef9-b87d-4993-a24e-8153377971c6.png)
We get no crash no error, it just doesn't work... I have the dll if you want!

Reply:
on a side note, I changed the way [we pass the indexes](https://github.com/DataDog/dd-trace-dotnet/blob/b9b2c8c95e0dc5c966184933b404f5d1ed7d06ba/tracer/src/Datadog.Tracer.Native/calltarget_tokens.cpp#L519) so that it remains more generic. 
that way debugger side, it just gives an empty array and nothin happens..

Reply:
@anna-git and I decided to avoid blocking this PR and to investigate if it's possible to consolidate the two new `System.Exception` locals at a later time.